README_utm_arm.txt

```diff
@@ -40,7 +40,7 @@ On an M1 Mac laptop, we cannot use virtual box, so follow these instructions ins
 
 
 3. Download and save the Ubuntu 20.04 ARM Server ISO
-   https://mac.getutm.app/gallery/ubuntu-20-04
+   https://mac.getutm.app/gallery/ubuntu-20-04 (click "Alternative and previous releases")
```

Comment:
Going to this URL, I could not see anything that said "Alternative and previous releases", and the ISO was under the "Downloads" header.

Reply:
I updated the instructions to clarify, thanks for pointing this out.